src/indexApi.ts

```diff
@@ -96,7 +98,9 @@ async function main(): Promise<void> {
     }
     let { start, end, appId, pluginIds, timePeriod } = analyticsQuery
     timePeriod = timePeriod.toLowerCase()
-    const queryStart = new Date(start).getTime() / 1000
+    let sevenDayOffset = 0
+    if (timePeriod === 'day') sevenDayOffset = SIX_DAYS_IN_MINUTES
```

Comment:
Avoid mutable variable when possible.
The fact that other parts of the code are "not great" doesn't mean you need to copy it ;)
Also, I don't like the variable name. Change to this:

```
const queryOffset = timePeriod === 'day' ? SIX_DAYS_IN_MINUTES : 0
```

Reply:
Made changes according to comment.